<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerSettings</class>
 <widget class="QWidget" name="PowerSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="tTitleLabel" name="titleLabel">
     <property name="text">
      <string>Power</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>725</width>
        <height>539</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignHCenter">
        <widget class="QWidget" name="powerProfileWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>POWER PROFILE</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>A power profile tweaks the way your system runs in order to prioritise performance or reduce power consumption.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="tStatusFrame" name="performanceInhibition"/>
          </item>
          <item>
           <widget class="QRadioButton" name="powerStretchProfileButton">
            <property name="text">
             <string>Power Stretch</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">powerProfilesGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Reduce the performance of your device to save on power</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="balancedProfileButton">
            <property name="text">
             <string>Balanced</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">powerProfilesGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Performance and power consumption will be average</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="performanceProfileButton">
            <property name="text">
             <string>Performance</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">powerProfilesGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="performanceDescription">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Performance will be increased at the cost of power consumption</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>1</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QWidget" name="timeoutsWidget" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>TIMEOUTS</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Switch off the screen after</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="screenOffSpin">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>If you leave your device idle for a period of time, we can automatically turn off the screen or suspend it for you.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QComboBox" name="screenOffUnit">
            <item>
             <property name="text">
              <string>seconds</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>minutes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>hours</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>never</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Suspend after</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="suspendSpin">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QComboBox" name="suspendUnit">
            <item>
             <property name="text">
              <string>seconds</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>minutes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>hours</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>never</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>1</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QWidget" name="buttonsWidget" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>When pressing the Power button</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="powerButtonActionBox">
            <item>
             <property name="text">
              <string>Ask</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Power Off</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Reboot</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Suspend</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Hibernate</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Do Nothing</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>BUTTONS</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tTitleLabel</class>
   <extends>QLabel</extends>
   <header location="global">ttitlelabel.h</header>
   <slots>
    <signal>backButtonClicked()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>tStatusFrame</class>
   <extends>QFrame</extends>
   <header location="global">tstatusframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="powerProfilesGroup"/>
 </buttongroups>
</ui>
